crml/transaction-payment/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,93 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2019-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Runtime API definition for transaction payment module.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::Codec;
+use sp_runtime::traits::MaybeDisplay;
+
+pub use crml_transaction_payment::{FeeDetails, InclusionFee, RuntimeDispatchInfo};
+
+// TODO Fix conflicting implementations of trait `parity_scale_codec::Decode` for type `pallet_transaction_payment::RuntimeDispatchInfo<_>`:
```

Comment:
will you open an issue or address in this PR? I think these changes should be added to `transaction-payment/src/types.rs`

Reply:
We have this related issue open here: https://github.com/plugblockchain/plug-blockchain/issues/180. I will check if your suggested solution there would address the issue here.

Reply:
Fixed it according to the suggestion.

Reply:
can clean up this file now

Reply:
The clean up is done as well.